site/docs/master/style-guide.md

```diff
@@ -0,0 +1,313 @@
+<tr><td>Plugin</td><td>Plugin rather than plug-in or other variations.</td></tr>
+</table>
+
+## Markdown elements
+
+
+### Line breaks
+Use a single newline to separate block-level content like headings, lists, images, code blocks, and others. The exception is second-level headings, where it should be two newlines. Second-level headings follow the first-level (or the title) without any preceding paragraphs or texts. A two line spacing helps visualize the overall structure of content in a code editor better.
+
+### Headings
+People accessing this documentation may use a screen reader or other assistive technology (AT). [Screen readers](https://en.wikipedia.org/wiki/Screen_reader) are linear output devices, they output items on a page one at a time. If there is a lot of content on a page, you can use headings to give the page an internal structure. A good page structure helps all readers to easily navigate the page or filter topics of interest.
+
+<table caption="Do and Don't - Headings" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td>Include a title on each page the page or blog post.</td><td>Use include more than one title headings (#) in a page.</td></tr>
+<tr><td>Use ordered headings to provide a meaningful high-level outline of your content.</td><td>Use headings level 4 through 6, unless it is absolutely necessary. If your content is that detailed, it may need to be broken into separate articles.</td></tr>
+<tr><td>Use sentence case for headings. For example, **Extend kubectl with plugins**</td><td>Use title case for headings. For example, **Extend Kubectl With Plugins**</td></tr>
+</table>
+
+### Paragraphs
+
+<table caption="Do and Don't - Paragraphs" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td>Try to keep paragraphs under 6 sentences.</td><td>Write long-winded paragraphs.</td></tr>
+<tr><td>Use three hyphens (`---`) to create a horizontal rule for breaks in paragraph content.</td><td>Use horizontal rules for decoration.</td></tr>
+</table>
+
+### Links
+
+<table caption="Do and Don't - Links" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td>Write hyperlinks that give you context for the content they link to. For example: Certain ports are open on your machines. See <a href="#check-required-ports">Check required ports</a> for more details.</td><td>Use ambiguous terms such as “click here”. For example: Certain ports are open on your machines. See <a href="#check-required-ports">here</a> for more details.</td></tr>
+<tr><td>Write Markdown-style links: `[link text](URL)`. For example: `[community meeting agenda](https://hackmd.io/Jq6F5zqZR7S80CeDWUklkA)` and the output is  [community meeting agenda](https://hackmd.io/Jq6F5zqZR7S80CeDWUklkA).</td><td>Write HTML-style links: `<a href="/media/examples/link-element-example.css" target="_blank">Visit our tutorial!</a>`, or create links that open in new tabs or windows. For example: `[example website](https://example.com){target="_blank"}`</td></tr>
```

Comment:
```suggestion
<tr><td>Write Markdown-style links: `[link text](URL)`. For example: `[community meeting agenda](https://hackmd.io/Jq6F5zqZR7S80CeDWUklkA)` and the output is  [community meeting agenda](https://hackmd.io/Jq6F5zqZR7S80CeDWUklkA).</td><td>Write HTML-style links: `<a href="/media/examples/link-element-example.css" target="_blank">Visit our tutorial!</a>`, or create links that open in new tabs or windows. For example: `[example website](https://example.com){target="_blank"})`</td></tr>
```

Reply:
FYI suggestions will not be accepted by our DCO checker